hypothesisfuzzer/fuzzing/repo_fuzzer.py

```diff
@@ -57,17 +59,20 @@ def get_errors():
         with open('data.txt', 'r') as file_data:
             return jsonify(json.load(file_data))
 
-    def _clone_git(self, git_url, folder_name="code"):
+    def _clone_git(self, git_url):
         # Delete old code folder
 
-        if os.path.exists(folder_name):
-            shutil.rmtree(folder_name, ignore_errors=True)
+        if os.path.exists(self.name):
+            shutil.rmtree(self.name, ignore_errors=True)
+
+        os.makedirs(self.name)
+        GitRepo.clone_from(git_url, self.name)
+
+    def _create_venv(self):
 
-        os.makedirs(folder_name)
-        GitRepo.clone_from(git_url, folder_name)
+        os.chdir(self.name)
+        assert os.path.basename(os.getcwd()) == self.name
```

Comment:
Why do we need this assertion?

Reply:
So we don't get anymore bugs with being in the wrong OS (if we do, we will know immediately).